timeline.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>PPoM - Timeline</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	
+</head>
+<body>
+	<!-- navbar -->
+<nav>
+	<a href="#home" class="nav-item">Home</a>
+	<a href="#timeline" class="nav-item">Timeline</a>
+	<a href="#skills" class="nav-item">Skills</a>
+	<a href="Eleanor Roberts CV.pdf" class="nav-item cv" target="_blank">CV</a>
+</nav>
+	<!-- header -->
+<header id="home">
+	<h1>History of Me</h1>
+</header>
+	<!-- timeline -->
+<div>
+	<section id="timeline">
+		<div class="timeline">
+		  <div class="container left">
+		    <div class="content">
```

Comment:
Think you can perhaps get rid of this div also so it is just in the container left div and format it all in there?